src/data.js

```diff
@@ -0,0 +1,48 @@
+import * as dom from './dom';
+
+export default async function fetchData() {
+  const apikey = 'bc92c530991c288d75d5cf2407507597';
```

Comment:
- [ ] Kindly use the ES6 arrow function instead of the function keyword as per the project [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png)